framework/tlvf/test/main.cpp

```diff
@@ -33,11 +36,66 @@ MAPF_INITIALIZE_LOGGER
 
 using namespace mapf;
 
+void dump_buffer(uint8_t *buffer, size_t len)
```

Comment:
Could you make a separate commit that moves out this function from the existing code?

It might be better to return a std::string and do the LOG part in the caller. That gives more flexibility for log level, and sets the correct line and file. Eventually, I think there should be an extension to easylogging++ for hexdumps. Something like `HEXLOG(INFO, buf, buf_len);`.